/**:
  ros__parameters:
    timeout_ms: 100
    # Topic names used here reflect former hardcoded ones in pure_pursuit_nodes.cpp
    command_topic: "ctrl_cmd"
    state_topic: "current_pose"
    tf_topic: "tf"
    static_tf_topic: "static_tf"
    trajectory_topic: "trajectory"
    diagnostic_topic: "ctrl_diag"
    controller:
      minimum_lookahead_distance: 6.0
      maximum_lookahead_distance: 100.0
      speed_to_lookahead_ratio: 2.0
      is_interpolate_lookahead_point: false
      is_delay_compensation: false
      emergency_stop_distance: 0.1
      speed_thres_traveling_direction: 0.3
      dist_front_rear_wheels: 2.7